packages/orbit-components/src/TileGroup/TileGroup.stories.tsx

```diff
@@ -14,272 +15,265 @@ import TileGroup from ".";
-        onClick={action("clicked")}
-        expandable
-        header={
-          <Stack justify="between" align="center" direction="row" shrink>
-            <Stack spacing="none" direction="column" shrink>
-              <Stack direction="row" align="center" spacing="200">
-                <Heading type="title4" as="h4">
-                  Mr. Hot potato
-                </Heading>
-                <CountryFlag code="cz" />
+              <Stack align="center" basis="0%">
+                <Badge type="infoSubtle">You</Badge>
               </Stack>
-              <Text>13/37/1337</Text>
-            </Stack>
-            <Stack align="center" basis="0%">
-              <Badge type="infoSubtle">You</Badge>
             </Stack>
-          </Stack>
-        }
-      >
-        This is example of expanded content
-      </Tile>
-      <Tile
-        icon={Icon && <Icon />}
-        onClick={action("clicked")}
-        expandable
-        header={
-          <Stack justify="between" align="center" direction="row" shrink>
-            <Stack spacing="none" direction="column" shrink>
-              <Stack direction="row" align="center" spacing="200">
-                <Heading type="title4" as="h4">
-                  Mr. Hot potato
-                </Heading>
-                <CountryFlag code="cz" />
+          }
+        >
+          This is example of expanded content
+        </Tile>
+        <Tile
+          icon={Icon && <Icon />}
+          onClick={action("clicked")}
+          expandable
+          header={
+            <Stack justify="between" align="center" direction="row" shrink>
+              <Stack spacing="none" direction="column" shrink>
+                <Stack direction="row" align="center" spacing="200">
+                  <Heading type="title4" as="h4">
+                    Mr. Hot potato
+                  </Heading>
+                  <CountryFlag code="cz" />
+                </Stack>
+                <Text>13/37/1337</Text>
+              </Stack>
+              <Stack align="center" basis="0%">
+                <Badge type="infoSubtle">You</Badge>
               </Stack>
-              <Text>13/37/1337</Text>
-            </Stack>
-            <Stack align="center" basis="0%">
-              <Badge type="infoSubtle">You</Badge>
             </Stack>
-          </Stack>
-        }
-      >
-        This is example of expanded content
-      </Tile>
-      <Tile
-        icon={Icon && <Icon />}
-        onClick={action("clicked")}
-        expandable
-        header={
-          <Stack justify="between" align="center" direction="row" shrink>
-            <Stack spacing="none" direction="column" shrink>
-              <Stack direction="row" align="center" spacing="200">
-                <Heading type="title4" as="h4">
-                  Mr. Hot potato
-                </Heading>
-                <CountryFlag code="cz" />
+          }
+        >
+          This is example of expanded content
+        </Tile>
+        <Tile
```

Comment:
There's code duplication of `Tile` component in each story. I would use `Array.from(...)` and get rid of redundant code.

Reply:
I did this for both Playground and RTL stories.